sdks/python/apache_beam/transforms/combiners.py

```diff
@@ -802,8 +803,8 @@ def add_input(self, accumulator, element, *args, **kwargs):
 
 class ToList(CombinerWithoutDefaults):
   """A global CombineFn that condenses a PCollection into a single list."""
-  def __init__(self, label='ToList'):  # pylint: disable=useless-super-delegation
-    super(ToList, self).__init__(label)
+  def __init__(self):  # pylint: disable=useless-super-delegation
```

Comment:
You could remove this constructor completely. The interpreter invokes `super().__init__()` if there's no constructor anyway. Pylint has also detected that this is meaningless. 

The same applies to `ToList` and `ToDict`